mmv1/third_party/terraform/services/datalossprevention/resource_data_loss_prevention_discovery_config_test.go

```diff
@@ -116,7 +116,10 @@ func testAccDataLossPreventionDiscoveryConfig_ActionsUpdate(t *testing.T) {
 	acctest.VcrTest(t, resource.TestCase{
 		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
 		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
-		CheckDestroy:             testAccCheckDataLossPreventionDiscoveryConfigDestroyProducer(t),
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"time": {},
```

Comment:
Why do we need the time provider here?

Reply:
Oops, forgot to remove that; attempted to make a test work by adding a wait, but it turned out using "depends_on" was better. Removed.